### PR TITLE
Task 793

### PR DESCRIPTION
**Pull request by msinansahin**
## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: devbuilds/linux/debian-64/control [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Version: @version
+Source: dagcoin
+Section: finance
+Priority: optional
+Maintainer: Dagcoin <support@dagcoin.org
```
Comment (yaryribero) [inline devbuilds/linux/debian-64/control]: Missing closing bracket?

File: devbuilds/linux/debian-64/copyright [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+This program is free software ; you can redistribute it
+[...]
+.
+On Debian systems, the full text of the GNU General Public
+License version 2 can be found in the file
+‘/usr/share/common-licenses/GPL-2’.
+Files: debian /*
+Copyright: Copyright 1998 Jane Smith <jsmith@example.net >
+License:
+[ LICENSE TEXT ]
```
Comment (yaryribero) [inline devbuilds/linux/debian-64/copyright]: We have license information: https://github.com/dagcoin/dagcoin/blob/devnet/LICENSE

Can you fill this file using it?

File: devbuilds/linux/debian-64/dagcoin.desktop [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Type=Application
+Version=@version
+Name=Dagcoin
+Comment=A wallet for decentralized value
+Exec=/usr/share/Dagcoin/Dagcoin
+Icon=/usr/share/Dagcoin/icon_256x256.png
+Terminal=false
+Categories=Office;Finance;
+MimeType=x-scheme-handler/dagcoin-tn;
```
Comment (yaryribero) [inline devbuilds/linux/debian-64/dagcoin.desktop]: Shouldn't depend on the environment?

File: public/views/modals/customized-amount.html [hunk 4]
```diff
@@ -14,8 +14,8 @@ <h4 class="title no-margin" translate>QR Code</h4>
       <div class="m10t text-center" ng-show="isCordova">
         <span class="button outline dark-gray tiny round"
           ng-click="shareAddress('Payment request: ' + amountInSmallestUnits / dagUnitValue + ' DAG to ' + addr)">
-          <i class="fi-share"></i>
           <span translate>Share address</span>
+          <i class="fi-share rotate-180"></i>
```
Comment (yaryribero) [inline public/views/modals/customized-amount.html]: Is this related to another ticket?

File: src/css/main.scss [hunk 5]
```diff
@@ -873,6 +873,10 @@ ul.manage li {
   color: #7a8c9e;
 }
 
+.rotate-180::before {
+  transform: rotate(180deg);
```
Comment (yaryribero) [inline src/css/main.scss]: Is this related to another ticket?

File: devbuilds/linux/debian-64/dagcoin.links [hunk 6]
```diff
@@ -0,0 +1 @@
+usr/share/Dagcoin/Dagcoin@type usr/bin/dagcoin
```
Comment (yaryribero) [inline devbuilds/linux/debian-64/dagcoin.links]: Is it possible to define different commands for testnet and live?
Tesnet would be dagcoin-tn
Livenet: dagcoin

File: public/views/modals/customized-amount.html (hunk 4 above)
Comment (msinansahin) [reply public/views/modals/customized-amount.html]: I will reload from head. How it was changed, I don't know.

File: src/css/main.scss (hunk 5 above)
Comment (msinansahin) [reply src/css/main.scss]: I will reload from head. How it was changed, I don't know.

File: devbuilds/linux/debian-64/dagcoin.links (hunk 6 above)
Comment (msinansahin) [reply devbuilds/linux/debian-64/dagcoin.links]: the type place-holder @type represents empty string or "-tn". The script is changing the dagcoin strings in files. The testnet executable file is being created as "dagcoin-tn"?

File: public/views/modals/customized-amount.html (hunk 4 above)
Comment (msinansahin) [reply public/views/modals/customized-amount.html]: This change is coming from merge with base branch, I think.

File: src/css/main.scss (hunk 5 above)
Comment (msinansahin) [reply src/css/main.scss]: This change is coming from merge with base branch, I think.
